doc/Doxyfile

```diff
@@ -2181,7 +2181,7 @@ PREDEFINED             = __attribute__(x)=
 # definition found in the source code.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-EXPAND_AS_DEFINED      =
+EXPAND_AS_DEFINED      = __READ_FUNC __WRITE_FUNC __FLUSH_FUNC
```

Comment:
Alright. I've added a `__MEMRW_` prefix so that the `EXCLUDE_SYMBOLS` pattern is shorter, and besides, this will reduce the risks of collisions with other headers.